src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -78,13 +78,19 @@ class BehandlerService(
             .map { (pBehandler, _) -> pBehandler.id }.firstOrNull() != pBehandler.id
 
         val behandlerIkkeKnyttetTilArbeidstaker = !pBehandlereForArbeidstakerList
+            .filter { (_, behandlerType) -> behandlerType == behandlerArbeidstakerRelasjon.type.name }
             .map { (pBehandler, _) -> pBehandler.id }.contains(pBehandler.id)
 
         if (isBytteAvFastlege || behandlerIkkeKnyttetTilArbeidstaker) {
             addBehandlerToArbeidstaker(
                 behandlerArbeidstakerRelasjon = behandlerArbeidstakerRelasjon,
                 behandlerId = pBehandler.id,
             )
+        } else if (behandlerArbeidstakerRelasjon.type == BehandlerArbeidstakerRelasjonType.SYKMELDER) {
+            updateBehandlerToArbeidstaker(
```

Comment:
Dette gjør at vi har et updatedAt-tidsstempel for sist gang vi har mottatt en sykmelding fra en lege for en gitt arbeidstaker.